src/commands/CmdInfo.cpp

```diff
@@ -44,8 +44,50 @@
 
 #include <algorithm>
 #include <sstream>
+#include <string>
 
 ////////////////////////////////////////////////////////////////////////////////
+
+static std::string wrapWithPrefixes(const std::string& text, size_t width,
+                                    const std::string& prefixFirst, const std::string& prefixCont) {
```

Comment:
There's existing wrapping support in src/libshared/src/shared.cpp, and I think that could be re-used for this purpose rather than re-implementing it.